doc/getting-started/launching-a-testnet.md

```diff
@@ -0,0 +1,103 @@
+### Prerequisites
+You must already be able to build everything in the `cardano-node` repository.
```

Comment:
Maybe link to the build instructions?

Reply:
Or to the environment setup instructions.

Reply:
I linked to `install.md`